examples/openspending/components/Header.tsx

```diff
@@ -16,18 +16,22 @@ export function Header() {
   };
 
   const navLinks = [
-    {
-      title: 'Home',
-      href: '/',
-    },
     {
       title: 'Datasets',
       href: '/#datasets',
     },
-    // {
-    //   title: "Community",
-    //   href: "https://community.openspending.org/"
-    // }
+    {
+       title: "Blog",
+       href: "/blog"
+    },
+    {
+       title: "About",
+       href: "/about"
+    },
+    {
+       title: "Resources",
```

Comment:
I think we are missing a couple of links e.g. about/funders, tools, presentations. Can we make the nav link a dropdown menu?

Reply:
Added a few, but note that 1. A LOT of the links/images/embeds are broken and:

- I didnt add the "About this website" section from the "About" since its not the same website and it wouldnt make sense to move that
- I've removed the handbook since its not a markdown file(seems to be some sort of python book) and i dont have the source code for that
- Removed a few of the resources things because of the markdowndb error that i'm getting(About trying to index too many pages at once) 

![image](https://github.com/datopian/portaljs/assets/11317382/4e430976-81a6-47dd-b7cf-76108d161b1a)
![image](https://github.com/datopian/portaljs/assets/11317382/32a69937-1f05-427d-93e0-7351dd282770)

Reply:
Done

Reply:
Makes sense. I see now that some pages really shouldn't be there. I think we have to remove IATI (it mentions an api that doesn't exist) and openspending.mobi (doesn't work)